src/com/zone5cloud/core/thirdpartyconnections/connections/ConectionApp.java

```diff
@@ -0,0 +1,20 @@
+package com.zone5cloud.core.thirdpartyconnections.connections;
+
+import com.google.gson.annotations.SerializedName;
+
+public class ConectionApp {
+
+	@SerializedName("company")
+	private ConnectionCompany company;
+
+	@SerializedName("user")
+	private ConnectionUser connectionUser;
```

Comment:
this should be in `ConnectionsResponse`

Reply:
Retrofit Good practices should avoid nested classes. It is better to use classes on its own.
the nested classes constructors trying to initialize the outer classes fields are a bit sketchy inside Retrofit converter factories.